src/job_cache/job_cache.cpp

```diff
@@ -219,14 +233,16 @@ FindJobResponse Cache::read(const FindJobRequest &find_request) {
       // TODO: Try to reconnect to the daemon, launching our own if need be.
       //       if that fails then depending on user preference either fail
       //       or return a cache miss.
-      log_fatal("Cache::read(): failed receiving message");
+      wcl::log::error("Cache::read(): failed receiving message");
+      exit(1);
```

Comment:
What if `wcl::log::fatal()` writes a message like `Unexpected fatal exit occured. See wake.log for details` to stderr?

Reply:
We don't expect (but in reality it'll happen) anyone to hit a log fatal so  its not like this would be a common case for anyone but us during development

Reply:
I would be fine with that but this isn't going to wake.log right now as this is client side. If we're going to multiplex to the wake.log on the client side for Cache::read() messages we should probably just print the actual error to stderr